appspec.yml

```diff
@@ -0,0 +1,20 @@
+version: 0.0
+os: linux
+files:
+ - source: /Buddies.Web/out/
+   destination: /var/www/html/
+ - source: /Buddies.API/bin/Release/net6.0/linux-x64/publish/
+   destination: /home/ec2-user/dotnetapi/
+file_exists_behavior: OVERWRITE
+hooks:
+  ApplicationStart:
+    - location: scripts/startapp.sh
+      timeout: 300
+      runas: root
+    - location: scripts/startapi.sh
+      timeout: 300
+      runas: root
+  ApplicationStop:
+    - location: scripts/stopapi.sh
+      timeout: 300
+      runas: root
```

Comment:
This file is ci/cd specific and it should only reside in that pr. This pr should only contain nginx related config.

Reply:
should i make a new branch and add the nginx stuff there, cause i branched this branch from the ci/cd branch so a lot of the non-nginx related files got included.

Reply:
This should have been branched off of main and only included the nginx config.